Workflows/Architecture review.md

```diff
@@ -26,9 +26,30 @@ Please follow this timeline:
 8. Once you've finished the database design, it has to be approved by someone from the management.
 9. Once your architecture has been approved, you can generate a new Rails project. Read more about that on the next page.
 
+Ask the project manager to give you the project specification.
+
+## Database design & discuss major implementation details
+Once you're done with the initial project meeting and have the specification, you can start designing the database. Use an online tool so you can easily present the diagram later on the review, such as [dbdiagram](https://dbdiagram.io/home) or [SqlDBM](https://sqldbm.com/Home/), or some other.
+
+Consult the project manager at anytime and add new discoveries to the specification if you find that something's missing. Don't hesitate to involve other team members since this is the most important part of the app.
+
+When you're done with the database diagram, sit down with the member of your team that was at the meeting with you and go through:
+* the database diagram
+* authentication
+ - what gems will be used
+ - API authentication - cookies (web frontend) or jwt/custom tokens (mobile frontend), or both
+* admin UI setup
+* emails - templates (preferred) or custom design (rarely a good use of time for MVP)
+* any 3rd party services
```

Comment:
We go in depth for some points (e.g. JWT/cookies), but we cast a very wide net here. I think we should be more specific - decide/reference decisions about APM, performance budgets, team collaboration, client AWS accounts, etc